### ISSUE TITLE
Create a PyTorch DataSet class to load our own data for Mask-RCNN

### ISSUE THREAD

**Issue by d-diaz** [d-diaz/stand_mapping#16]
PyTorch's default Mask-RCNN [implementation ](https://pytorch.org/docs/stable/torchvision/models.html#torchvision.models.detection.maskrcnn_resnet50_fpn) expects input formatted like this:

> During training, the model expects both the input tensors, as well as a targets (list of dictionary), containing:
> - boxes (`FloatTensor[N, 4]`): the ground-truth boxes in `[x1, y1, x2, y2]` format, with values of `x` between `0` and `W` and values of `y` between `0` and `H`
> - labels (`Int64Tensor[N]`): the class label for each ground-truth box
> - masks (`UInt8Tensor[N, H, W]`): the segmentation binary masks for each instance

We will probably want to implement a DataSet class to make data loading and any transformations straightforward. Here's a [tutorial ](https://pytorch.org/tutorials/beginner/data_loading_tutorial.html) in PyTorch docs on making a DataSet. Here is an [example ](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)specific to creating a DataSet for Mask-RCNN .
